src/views/Editor.vue

```diff
@@ -87,6 +87,18 @@
 					<b-btn v-b-tooltip.hover title="Publish makes the saved dataset public. Remember to always save the datset before publishing (only the latest saved version gets published)." @click="confirmPublish" :disabled="rateLimited" ref="dataset-publish-button">Publish</b-btn>
 				</b-button-group>
 			</div>
+
+			<div v-else :style="{'padding': '20px'}">
+				Please select one option from "Where are my files" menu. Note that the selected option cannot be changed without creating a new dataset.
+				<br>
```

Comment:
instead of having two br, should we have p tags around those chapters.

Reply:
True. I just moved this text from another file without changing the content.

Reply:
changed